tools/crm_mon.c

```diff
@@ -254,9 +261,9 @@ apply_include(const gchar *includes, GError **error) {
             show |= mon_show_times;
         } else {
             g_set_error(error, G_OPTION_ERROR, CRM_EX_USAGE,
-                        "--include options: all, attributes, bans, counts, dc, default, "
-                        "defaults, failcounts, fencing, nodes, none, operations, options, "
-                        "resources, stack, summary, tickets, times");
+                        "--include options: all, attributes, bans[:PREFIX], counts, dc, "
+                        "default, defaults, failcounts, fencing, nodes, none, operations, "
```

Comment:
We don't need to have both default and defaults in the help text, we can pick one as the "right" one and silently support the other for anyone who remembers it wrong